.github/workflows/ci.yml

```diff
@@ -110,7 +114,8 @@ jobs:
         with:
           path: '**/node_modules'
           key:
-            test-community-modules-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+            test-community-modules-${{ runner.os }}-${{ hashFiles('yarn.lock')
+            }}
```

Comment:
We can probably remove this since David has a followup PR to remove the community package.

Reply:
removed it as to align with David's PR
https://github.com/carbon-design-system/ibm-products/pull/6241/files#